src/resources/learningResource/learningResource.model.js

```diff
@@ -0,0 +1,35 @@
+import mongoose from 'mongoose';
+
+const learningResourceSchema = new mongoose.Schema({
+  category: {
+    type: String,
+    enum: ['Coding', 'Maths', 'Electronics'],
+    required: true
+  },
+  minAge: {
+    type: Number,
+    required: true
+  },
+  maxAge: {
+    type: Number,
+    required: true
+  },
```

Comment:
Do we need to require ages? What about 'All ages' cases or no max age?

Reply:
In the resource form(from figma) age field is mandatory, so it should be fine.

Reply:
Hmm... I obviously hadn't looked properly at the design. The 'age' field is a single drop-down menu. If these were just a set of hard-coded options in the form, perhaps it would make more sense for this field to be a single string rather than two numbers. That might also allow the simple inclusion of 'all ages' as an option, but it's potentially a bit limiting as well.

Happy to cross that bridge when we come to it, though.

It would be amusing to attend an event with a minimum age of 3 but no maximum age...

Reply:
Changed my mind.

I think based on the design it makes sense for the field to be a string, but mandatory. Then the options can be specified in the form. Shall edit.

Reply:
Ah yeah makes sense, sorry I didn't actually check the design in Figma! 🤦‍♂️

Yeah we can update it later if need be 👍